Personal Access Token documentation updates (#572)
rudderlabs/rudderstack-docs: data-governance/tracking-plans/index.mdx

To start using the Tracking Plans, follow these steps:

> 1. Get your [Personal Access Token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Author: ```suggestion
1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.
```

> 1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Reviewer: Should change to

> **Generate a** [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) **in** the RudderStack dashboard.

> 1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Author: ```suggestion
1. Generate a [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) in the RudderStack dashboard.
```

To use the Tracking Plan API, follow these steps:

> 1. Get your [Personal Access Token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Author: ```suggestion
1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.
```

> 1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Reviewer: Should change to

> **Generate a** [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) **in** the RudderStack dashboard.

> 1. Get your [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) from the RudderStack dashboard.

Author: ```suggestion
1. Generate a [personal access token](https://www.rudderstack.com/docs/rudderstack-api/personal-access-tokens/) in the RudderStack dashboard.
```
